app/services/scheduling_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class SchedulingService
+  DEFAULT_TIME = "9:00am"
+  DEFAULT_DATE = Time.zone.now.tomorrow
```

Comment:
I think we can't make this a constant, as it'll start being in the past :-|.

Reply:
This is now set as a variable in the view.